src/trailer.c

```diff
@@ -0,0 +1,415 @@
+/*
+ * Copyright (C) the libgit2 contributors. All rights reserved.
+ *
+ * This file is part of libgit2, distributed under the GNU GPL v2 with
+ * a Linking Exception. For full terms see the included COPYING file.
+ */
+#include "common.h"
+#include "git2/message.h"
+
+#include <stddef.h>
+#include <string.h>
+#include <ctype.h>
+
+static const char comment_line_char = '#';
+
+static const char *const git_generated_prefixes[] = {
+	"Signed-off-by: ",
+	"(cherry picked from commit ",
+	NULL
+};
+
+static const char *const separators = ":";
+
+static inline int is_blank_line(const char *str)
```

Comment:
`inline` needs to be `GIT_INLINE`

Reply:
I might just drop the `inline` entirely. It's a small static function so it'll probably end up being inlined automatically anyway.